doc/infrastructure.md

```diff
@@ -294,56 +292,79 @@ Started Apr. 2020
 
 Maintainer: Vaclav Petras
 
-* <https://github.com/OSGeo/grass/actions>
-* Details: <https://github.com/OSGeo/grass/pull/525>
-
-* CI workflow with:
-  * A build job which is not parallelized and is meant for clear & relatively fast
+- <https://github.com/OSGeo/grass/actions>
+- Details: <https://github.com/OSGeo/grass/pull/525>
+- CI workflow with:
+  - A build job which is not parallelized and is meant for clear & relatively fast
     check of compilation and building in general. (Duplicating what is running
     on Travis)
-  * A test job which of course needs to build, but the main focus is to run tests,
+  - A test job which of course needs to build, but the main focus is to run tests,
     so the compilation is parallelized (depending on nproc) and thus potentially
     less readable. This runs the whole test suite. (You need to run it locally to
     see the actual error, but you can see which tests are failing.)
-* Static code analysis/Code quality check using Flake8 with separate checks for
+- Static code analysis/Code quality check using Flake8 with separate checks for
   python/grass, gui/wxpython, scripts and temporal directories.
-  * Configurations ignore different lists of Flake8 errors. The idea is to reduce
+  - Configurations ignore different lists of Flake8 errors. The idea is to reduce
     that to minimum.
-  * Code in testsuite directories is also ignored for now, but should not be in
+  - Code in testsuite directories is also ignored for now, but should not be in
     the future.
 
-Helper files placed to .github/workflows
+Helper files placed in .github/workflows/
 
 ## GRASS Coverity Scan
 
 Maintainer: Markus Neteler
 
-* <https://scan.coverity.com/projects/grass?tab=overview>
+- <https://scan.coverity.com/projects/grass?tab=overview>
+
+## User message translation management (i18N)
+
+Messages are extracted with `gettext` message macros.
+
+Translations may be done using the OSGeo Weblate platform:
+
+- Weblate: <https://weblate.osgeo.org/>
+  - GRASS GIS Weblate server: <https://weblate.osgeo.org/projects/grass-gis/>
+
+For technical background on the Weblate installation, see <https://wiki.osgeo.org/wiki/SAC:Weblate>
+
+### How it works
+
+When a developer makes a GRASS GIS repo commit on GitHub, GitHub then calls
+the webhook on Weblate which triggers a refresh of Weblate's git copy of the
+GRASS GIS repo.
+
+For pushing translations back to GitHub, there is a setting in Weblate for that
+which defaults to 24 hrs (accumulates translations over a day). Then a pull
+request with the translations will be opened in the GRASS GIS GitHub repo.
+
+### Troubleshooting
+
+In case the Weblate's git copy of the GRASS GIS repo does not update due to
+a conflict or whatever reason:
 
-## Transifex translation management
+Log into Weblate and check on <https://weblate.osgeo.org/projects/grass-gis/#repository>
```

Comment:
I think the information on who can do that is missing. (Sort of a general issue here actually.) For example, "people approved by XY".

Reply:
I think anyone with OSGeo-LDAP access can translate.

Reply:
The context here is managing in case of issues, so that's what I had in mind. I logged into Weblate and I have Manage, but I have only Translation memory under that, so I guess special privilege needs to be granted for this, so back to my original questions.

Reply:
I am afraid I don't fully understand your comment.

Note that I have briefly documented the Weblate usage here (which is beyond the document scope here):
https://grasswiki.osgeo.org/wiki/GRASS_messages_translation#Weblate_translation_platform

Reply:
I see that the troubleshooting is explained at the wiki, so just delete the sentence from here and we can be done.

My original comment about permissions will then apply to the wiki. Not everyone has access to the "Repository Maintenance" menu item.

To clarify, I was not concerned about documenting translation process here, but about permissions needed for the case of troubleshooting which the new text included.

Reply:
I prefer to keep it and have added that Weblate troubleshooting requires administrator rights (e7442f38102ffd33664da920ad5e8c8a379ff4ed).

Reply:
Given that it is duplication of the wiki and more detail then in the other cases, I don't know if the text will survive in this file.

However, what is most relevant to the infrastructure is who has the rights or who can give them. This would improve the bus factor.

Reply:
Of course it will survive:

- we talk about 2 lines which explain access and user rights
- it explains where to find more technical background.

Not sure what we are discussing here.

Reply:
> However, what is most relevant to the infrastructure is who has the rights or who can give them. This would improve the bus factor.

Exactly: For posterity, this is the original SVN commit message:

```
------------------------------------------------------------------------
r20666 | markus | 2006-08-04 16:11:37 +0200 (Fri, 04 Aug 2006) | 2 lines

explained how it all works (just in case)
```

Reply:
I'm sorry, I'll try to clarify. Basically, I think the current text either says not enough, if we want to keep it, or it says too much and then we can reduce it.

Missing information: What is most relevant to the infrastructure is who has the rights or who can give them. Adding this information would improve the bus factor. I was suggesting to either add that or skip the whole not about troubleshooting which is duplicated from the wiki page. (Bad bus factor: unclear who has the right to do the management. Good bus factor: required rights are stated, who has the rights is known, who can give the right is known.)

Duplication: One can somehow deduce the admin stuff from the wiki page if they follow the link to https://wiki.osgeo.org/wiki/SAC:Weblate. My conclusion was that it is the OSGeo SAC  admins who can do that. If that's correct, the wiki says it all. No need to discuss it here more especially if no additional details are provided comparing to the wiki. Hence my note that I don't see why it should survive next "refactoring" of this file.

Looking at this again, just link both of the wiki pages and you can delete most of the Weblate-related text here. (I think only the SAC one is linked.)

Reply:
> Exactly...

Oh, yes. The term _bus factor_ makes one little uncomfortable just like your commit message.

Reply:
Since the other day https://wiki.osgeo.org/wiki/SAC:Weblate has been updated I am now fine with the current short version related to Weblate.

Reply:
Looks good to me!